File: BackEnd/BackEnd/Domain/Models/RespuestaCuestionario.cs
using Microsoft.EntityFrameworkCore.Metadata.Internal;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace BackEnd.Domain.Models
{
    public class RespuestaCuestionario
    {
        public int Id { get; set; }

        [Required]
        [Column(TypeName = "varchar(100)")]
        public string Participante { get; set; }

        public DateTime Fecha { get; set; }

        public int Activo { get; set; }


        public int CuestionarioId { get; set; }
        public Cuestionario? Cuestionario { get; set; }


        public List<RespuestaCuestionarioDetalle> ListCuestionarioDetalle { get; set; }

    }
}

File: App-Preguntas-Respuestas/src/app/models/RespuestaCuestionario.ts
import { RespuestaCuestionarioDetalle } from "./RespuestaCuetrionarioDetalle";

export class RespuestaCuestionario {


    cuestionarioId:number;
    participante:string;
    listCuestionarioDetalle:RespuestaCuestionarioDetalle[];
    


    /**
     *
     */
    constructor() {
      this.cuestionarioId=0;
      this.participante='';
      this.listCuestionarioDetalle = [];

    }





}